### ISSUE TITLE
value-list-max-empty-lines no worky

### ISSUE THREAD

**Issue by benfrain** [stylelint/stylelint#1768]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug with message: Undefined rule "value-list-max-empty-lines" as described here: https://github.com/stylelint/stylelint/blob/master/src/rules/value-list-max-empty-lines/README.md

**EDIT:** Wondering if this is described in the docs but the rule is not actually in current release?

> Which rule, if any, is this issue related to?

"value-list-max-empty-lines": 0,

> What CSS is needed to reproduce this issue?

Just won't run with this config present

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "value-list-max-empty-lines": 0
  }
}

```

> Which version of stylelint are you using?

7.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Not related to CSS

> What did you expect to happen?

The rule to run

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
[10:24:59] Error in plugin 'run-sequence(lint-styles)'
Message:
    Undefined rule "value-list-max-empty-lines"
Details:
    code: 78
```

**Comment by m-allanson** [comment]
Hi @benfrain, thanks for the detailed report. You're right that `value-list-max-empty-lines` [hasn't been released yet](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md). 

The docs here on GitHub represent the state of the master branch, the docs for the latest published release of stylelint can be found at http://stylelint.io.

I think this is an issue that's come up before, maybe there's a way for us to steer people towards the stylelint.io docs?

**Comment by evilebottnawi** [comment]
@m-allanson maybe add to `ISSUE_TEMPLATE.md` first note about this?

**Comment by m-allanson** [comment]
@evilebottnawi That's probably the most visible place to put it, but I wonder if we should only add things to `ISSUE_TEMPLATE.md` if it applies to the majority of issues?

We could add an item to the FAQs, although that is less visible than the issue template.

**Comment by evilebottnawi** [comment]
@m-allanson maybe add `FAQs` and add link in `ISSUE_TEMPLATE.md` ( and write `Before creating issue please read FAQs` or something like that).

**Comment by m-allanson** [comment]
@evilebottnawi That sounds good to me.

**Comment by davidtheclark** [comment]
Yep, diagnosis sounds right.
